concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -107,6 +107,13 @@ resources:
       region_name: {{aws_region}}
       versioned_file: bosh-CA.tar.gz
 
+  - name: logsearch-cert
```

Comment:
You've probably seen this, but the [team manual](https://government-paas-team-manual.readthedocs.org/en/latest/team/working_practices/#commit-messages) and [style guide](https://github.com/alphagov/styleguides/blob/master/git.md) expand on what @keymon is saying